doc/modules/configuration_and_utils/deduplicate_categorical_data.rst

```diff
@@ -47,3 +47,59 @@ dtype: object
 'white', 'white', 'white', 'white', 'white']
 
 See the |deduplicate| documentation for caveats and more detail.
+
+Deduplicating values in a dataframe
+-----------------------------------
+
+|deduplicate| can be used to replace values in a dataframe that contains typos.
+>>> import pandas as pd
+>>> df = pd.DataFrame({'color': duplicated, 'value': range(10)})
+>>> df
+color  value
+0  blacs      0
+1  black      1
+2  black      2
+3  black      3
+4  black      4
+5  uhibe      5
+6  white      6
+7  white      7
+8  white      8
+9  white      9
+>>> df['deduplicated_color'] = df['color'].map(deduplicate_correspondence.to_dict())
+>>> df
+color  value deduplicated_color
+0  blacs      0              black
+1  black      1              black
+2  black      2              black
+3  black      3              black
+4  black      4              black
+5  uhibe      5              white
+6  white      6              white
+7  white      7              white
+8  white      8              white
+9  white      9              white
+
+With polars:
+>>> import polars as pl  # doctest: +SKIP
```

Comment:
The skip doctest here is leading the line below to fail.

Maybe the line below should also be skipped?